tests/unittest.py

```diff
@@ -511,30 +534,36 @@ async def run_bg_updates():
 
         return hs
 
-    def pump(self, by=0.0):
+    def pump(self, by: float = 0.0) -> None:
         """
         Pump the reactor enough that Deferreds will fire.
         """
         self.reactor.pump([by] * 100)
 
-    def get_success(self, d, by=0.0):
-        deferred: Deferred[TV] = ensureDeferred(d)
+    def get_success(
+        self,
+        d: Awaitable[TV],
+        by: float = 0.0,
+    ) -> TV:
+        deferred: Deferred[TV] = ensureDeferred(d)  # type: ignore[arg-type]
```

Comment:
What does mypy want the type of `ensureDeferred` to be?

Reply:
Seems it was a `Coroutine` or `Deferred`: https://github.com/twisted/twisted/blob/5c24e99e671c4082a1ddc8dbeb869402294bd0dc/src/twisted/internet/defer.py#L1094-L1096

Reply:
yeah. We end up passing in things here that implement `Coroutine["Deferred[_T]", object, "_T"]`, (so they work in practice) but aren't typed that way (they just implement `Awaitable[_T]`). To keep the tests themselves clean, it's easier just to `ignore[arg-type]`.

Reply:
That makes sense, thanks!